.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -38,6 +38,12 @@ body:
       label: Relevant Logs
       description: If possible, reproduce the bug with the envrionment variable RUST_LOG set to 'trace', and attach the printed logs
       render: shell
+  - type: textarea
+    id: intproxy-logs
+    attributes:
+      label: Internal proxy logs
+      description: Please attach the relevant `mirrord-intproxy` logs. You may find them in your `/tmp` directory (default).
```

```suggestion
      description: Please attach the relevant `mirrord-intproxy` logs. You can find them in your `/tmp` directory (default).
```
Could you also add an example file name and mention that dst file can be configured?

---

Changed and added an example.